test/functional/page_objects/settings_page.ts

```diff
@@ -486,6 +486,46 @@ export function SettingsPageProvider({ getService, getPageObjects }: FtrProvider
       await this.clickSaveScriptedField();
     }
 
+    async addRuntimeField(name: string, type: string, script: string) {
+      await this.clickAddField();
+      await this.setFieldName(name);
+      await this.setFieldType(type);
+      if (script) {
+        await this.setFieldScript(script);
+      }
+      await this.clickSaveField();
+      await new Promise((resolve) => setTimeout(resolve, 1000 * 30));
```

Comment:
Do we really have to wait 30 seconds to make sure a field is saved? 🤔 I would have thought that it is the consumer who (1) add a runtime field by calling this function (2) wait inside a loop (with a limited time) until the field appears in a table.

Reply:
This was a failed tempt to address something thats happening in CI but not locally. Going back over it.